### ISSUE TITLE
memoria mod glitch

### ISSUE THREAD

**Issue by spkpal** [Albeoris/Memoria#983]
hey Mod people

when i tried to use the memoria mod today with the playable character mod with it

this happen in this screenshot

![ff9 memoria game glitch using the memoria mod manager](https://github.com/user-attachments/assets/d46cba03-badf-46d6-8754-e8650ec7d449)

thanks

**Comment by WarpedEdge** [comment]
Sorry,

What exactly is the issue? Is it that Blank's model is seen from the right?
Confused on how PCP would be causing this.

**Comment by Tirlititi** [comment]
Indeed, the issue is that a part of Blank's model is shown on the right part of the screen, while it should be hidden because he's out of the vision range (because that scene isn't widescreen). It's unrelated to the PCP mod, yes. But it's better to provide more informations than needed rather than fewer.

**Comment by Emeiru** [comment]
I'll more or less assume this is because of widescreen being enabled. I can remember a few scenes where actors are seen visible just like this, usually at the farthest end of the screen which should have been hidden in the original resolution.

This is in no way a problem with PCP, maybe the OP just mentioned it as one of the mods they have active.
